sys/arm64/conf/GENERIC-MORELLO-BENCHMARKABI

```diff
@@ -0,0 +1,25 @@
+#
+# GENERIC-MORELLO-BENCHMARKABI -- Generic kernel configuration file for CheriBSD/Morello benchmark ABI kernel
+#
+# For more information on this file, please read the config(5) manual page,
+# and/or the handbook section on Kernel Configuration Files:
+#
+#    https://www.FreeBSD.org/doc/en_US.ISO8859-1/books/handbook/kernelconfig-config.html
+#
+# The handbook is also available locally in /usr/share/doc/handbook
+# if you've installed the doc distribution, otherwise always see the
+# FreeBSD World Wide Web server (https://www.FreeBSD.org/) for the
+# latest information.
+#
+# An exhaustive list of options and more detailed explanations of the
+# device lines is also present in the ../../conf/NOTES and NOTES files.
+# If you are in doubt as to the purpose or necessity of a line, check first
+# in NOTES.
+#
+# $FreeBSD$
+
+include "GENERIC-MORELLO-PURECAP"
+
+ident		GENERIC-MORELLO-BENCHMARKABI
```

Comment:
I don't love the name for two reasons: (a) it doesn't say purecap anywhere (b) we don't put ABI in the other names even though that's what they are

Reply:
I initially called it `GENERIC-MORELLO-BENCHMARK` but was afraid that people would confuse the `GENERIC-MORELLO-BENCHMARK` and `GENERIC-MORELLO-BENCHMARK-NODEBUG`, which is actually the thing you want if you run benchmarks. I'm open to better names though.

Reply:
Maybe `GENERIC-MORELLO-PURECAP-BENCHMARK`? It remains confusing for the `-NODEBUG` version but there isn't much we can do I guess..

Reply:
Yeah, I was thinking something like that would be consistent, but I take your point...

Reply:
Updated it to use the new name. It will be confusing either way, so at least we are being consistent.